plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -168,6 +169,24 @@ class Bazel(
             }
         }
 
+    override fun findPackageManagerDependencies(
+        analysisRoot: File,
+        managedFiles: Map<PackageManager, List<File>>,
+        analyzerConfig: AnalyzerConfiguration
+    ): PackageManagerDependencyResult {
+        // If both Bazel and Conan are enabled, they should not run in parallel, because this may cause conflicts with
+        // the Conan HOME files (see https://github.com/oss-review-toolkit/ort/pull/10357). Therefore, Conan is always
+        // run before Bazel.
+        val conanFactory = analyzerConfig.getConanFactory()
+        val mustRunBefore = if (conanFactory != null) {
+            setOf(conanFactory.descriptor.id)
+        } else {
+            emptySet()
+        }
+
+        return PackageManagerDependencyResult(mustRunBefore = mustRunBefore, mustRunAfter = emptySet())
```

Comment:
IIUC, this does the opposite of what is documented above: It configures that "this" (= Bazel) `mustRunBefore` Conan. Am I getting the logic right, @mnonnenmacher?

Reply:
Yes you're right. Fixed.